src/app/Food.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package app;
+
+import javafx.scene.control.DatePicker;
+
+/**
+ *
```

Comment:
Please include a brief description of the class and anything special that needs to be done to use it in this comment block. Haynes will definitely grade on that.